eurekatrees.py

```diff
@@ -81,11 +81,15 @@ def get_js_struct(self, node, node_dict=None):
             node_dict = {'name': node.data, 'children': []}
         if node is not None:
             if node.left:
-                new_node_dict_left = {'name': node.left.data, 'children': []}
+                new_node_dict_left = {'name': node.left.data, 'type':'left', 'is_prediction':False 'children': []}
                 node_dict['children'].append(self.get_js_struct(node.left, new_node_dict_left))
             if node.right:
-                new_node_dict_right = {'name': node.right.data, 'children': []}
+                new_node_dict_right = {'name': node.right.data, 'type':'right', 'is_prediction':False 'children': []}
```

Comment:
Needs comma after 'is_prediction':False. This is an error that needs to be fixed before I can accept your pull request.